src/briefcase/console.py

```diff
@@ -118,6 +162,67 @@ def error(self, message="", *, prefix="", markup=False):
         """Log message at error level; always included in output."""
         self._log(prefix=prefix, message=message, markup=markup, style="bold red")
 
+    def capture_stacktrace(self):
+        """Extract Rich stacktrace from exception while in except block."""
+        self.stacktrace = Traceback.extract(*sys.exc_info(), show_locals=True)
+
+    def save_log_to_file(self, command):
+        """Save the current application log to file."""
+        # only save log if a command ran and it errored or --save-log passed
+        if command is None or (not self.stacktrace and not command.save_log):
+            return
+
+        with command.input.wait_bar("Saving application log...", transient=True):
+            self.cleanup_log_files(command.logs_path)
```

Comment:
I originally had this happening every time briefcase ran.....but that seemed like unnecessary overhead.....since I'm mostly only concerned about many log files piling up over time. So, if log files being created often, then the check will run more often.

Reply:
Closing since users will be expected to manage log files making this moot.